charts/k8tz/values.yaml

```diff
@@ -13,6 +13,9 @@ verbose: false
 # Labels to apply to all resources
 labels: {}
 
+skipInjectNamespaces:
```

Comment:
all possible values should be documented in the [chart README.md values section](https://github.com/k8tz/k8tz/blob/master/charts/k8tz/README.md#values)

Reply:
also, I am wondering about the name `skipInjectNamespaces`, what about `webhook.ignoreNamespaces` or `webhook.skipNamespaces`?

Reply:
Thanks.  +1 on `ignoredNamespaces`:  Namespaces to be ignored when injecting the timezone.

Reply:
I changed the option to `webhook.ingoredNamespaces`

Reply:
Thanks. Please also add this new param in the [chart README.md values section](https://github.com/k8tz/k8tz/blob/master/charts/k8tz/README.md#values)